README.md

```diff
@@ -35,20 +35,31 @@ pip install intel-extension-for-transformers
 > For more installation methods, please refer to [Installation Page](./docs/installation.md)
 
 ## 🌟Introduction
-Intel® Extension for Transformers is an innovative toolkit to accelerate Transformer-based models on Intel platforms, in particular, effective on 4th Intel Xeon Scalable processor Sapphire Rapids (codenamed [Sapphire Rapids](https://www.intel.com/content/www/us/en/products/docs/processors/xeon-accelerated/4th-gen-xeon-scalable-processors.html)). The toolkit provides the below key features and examples:
+Intel® Extension for Transformers is an innovative toolkit designed to optimize the performance of Transformer-based models on various Intel platforms, including Intel Gaudi2, Intel CPU, and Intel GPU (TBD). The toolkit provides the below key features and examples:
```

Comment:
designed to accelerate GenAI/LLM everywhere with the optimal performance of Transformer-based models on various xxx